packages/mds-jurisdiction-service/client/index.ts

```diff
@@ -14,9 +14,17 @@
     limitations under the License.
  */
 
+import { UnwrapServiceResult, ServiceClient } from '@mds-core/mds-service-helpers'
 import { JurisdictionServiceProvider } from '../service/provider'
-import { JurisdictionServiceInterface } from '../@types'
+import { JurisdictionService } from '../@types'
 
-const { initialize, shutdown, ...client } = JurisdictionServiceProvider
+const { initialize, shutdown, ...service } = JurisdictionServiceProvider
 
-export const JurisdictionServiceClient: JurisdictionServiceInterface = client
+export const JurisdictionServiceClient: ServiceClient<JurisdictionService> = {
+  createJurisdiction: UnwrapServiceResult(service.createJurisdiction),
+  createJurisdictions: UnwrapServiceResult(service.createJurisdictions),
+  deleteJurisdiction: UnwrapServiceResult(service.deleteJurisdiction),
+  getJurisdiction: UnwrapServiceResult(service.getJurisdiction),
+  getJurisdictions: UnwrapServiceResult(service.getJurisdictions),
+  updateJurisdiction: UnwrapServiceResult(service.updateJurisdiction)
+}
```

Comment:
Man, I would _love_ if there was a concise way to do this. Create a new object with the same methods as another, but applies a wrapper over them.

Reply:
Totally. I spent a bit of time trying, but opted to establish the boundary between service/consumer and then trying again in a later PR. Would love to land on something akin to:
```ts
asServiceClient<I>(provider: ServiceProvider<I>): ServiceClient<I>
```